crates/cli-tools/src/changelog.rs

```diff
@@ -91,7 +97,7 @@ impl Changelog {
     }
 
     /// Parses and validates a changelog.
-    fn parse(input: &str) -> Result<Changelog> {
+    fn parse(crate_path: &str, input: &str) -> Result<Changelog> {
```

Comment:
nit: Since we need ownership, we can take `String` directly.
```suggestion
    fn parse(crate_path: String, input: &str) -> Result<Changelog> {
```

If we believe it's not convenient to call it anymore, we can take `impl Into<String>` instead, and do `let crate_path = crate_path.into()` at the beginning of the function.

Reply:
Using `impl Into<String>` for now